packages/components/CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### [Unreleased]
 
+#### Fixed
+
+- `Content Wrapper` bug in `@porsche-design-system/components-react/ssr` where an Error was thrown when children are
```

Comment:
I thought this wasn't only related to content wrapper, but could happen elsewhere, but only in SSG context?

Reply:
It was reported as Content Wrapper bug, so i thought it would help the consumer to make a connection between the changelog entry and the reported bug. I will concretize it.